server/testdb/runs.go

```diff
@@ -56,6 +56,21 @@ func (td *postgresDB) UpdateRun(ctx context.Context, r model.Run) error {
 	return nil
 }
 
+func (td *postgresDB) DeleteRun(ctx context.Context, r model.Run) error {
+	stmt, err := td.db.Prepare("DELETE FROM runs WHERE id = $1")
```

Comment:
Aren't there any foreign keys we should worry about?

Reply:
No, no entity has dependency with runs. It's the "last in the chain". Also, since we reference it by runID, we don't have to care about test version

Reply:
No thing relies on runs. We would have to worry about that if we were deleting tests or definitions.

Reply:
Cool, thanks!